### ISSUE TITLE
Why conv1 in first block is not followed by batch norm and relu

### ISSUE THREAD

**Issue by ghost** [wenxinxu/resnet_in_tensorflow#26]
https://github.com/wenxinxu/resnet-in-tensorflow/blob/8ba8d8905e099cd7e1b1cf1f84b89f603f7613a0/resnet.py#L136

Can you please highlight the necessary section in the paper or in the original implementation.

Thank you
